pocs/scheduler/scheduler.py

```diff
@@ -274,6 +276,14 @@ def read_field_list(self):
                 except Exception as e:
                     self.logger.warning("Error adding field: {}", e)
 
+    def set_common_properties(self, time):
+
+        self.common_properties = {
```

Comment:
Not really part of this PR, but since you are right here can you change this horizon to pull from the config rather than be hard-coded?

```python
horizon_limit = self.config['location'].get('observe_horizon', -18 * u.degree)      
self.common_properties = {
    'end_of_night': self.observer.tonight(time=time, horizon=horizon_limit)[-1],
    'moon': get_moon(time, self.observer.location),
    'observed_list': self.observed_list
}
```